lib/const.rb

```diff
@@ -1,16 +1,19 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.8.9'
+CREW_VERSION = '1.8.10'
 
-ARCH_ACTUAL = `uname -m`.strip
+ARCH_ACTUAL = `uname -m`.chomp
 # This helps with virtualized builds on aarch64 machines
 # which report armv8l when linux32 is run.
-ARCH = if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+ARCH = ENV['CREW_ARCH'] || if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+
+CREW_LIB_SUFFIX = if ARCH == 'x86_64' then '64' else '' end
```

Comment:
This would also work:
```
CREW_LIB_SUFFIX = (ARCH == 'x86_64') ? '64' : ''
```

Reply:
I think what we have right now makes it more readable.

Reply:
Done